.github/workflows/publish_docs_to_wiki.yml

```diff
@@ -16,23 +16,26 @@ env:
 
 jobs:
   publish_docs_to_wiki:
+    name: Publish docs to Wiki
     runs-on: ubuntu-latest
-    steps:  
-    - uses: actions/checkout@v2
+    timeout-minutes: 30
+    steps:
+      - name: Checkout source code
+      - uses: actions/checkout@v2
 
-    - name: Pull wiki
-      run: |
-         mkdir tmp_wiki
-         cd tmp_wiki
-         git init
-         git config user.name $USER_NAME
-         git config user.email $USER_EMAIL
-         git pull https://$USER_TOKEN@github.com/$OWNER/$REPOSITORY_NAME.wiki.git
+      - name: Pull Wiki
+        run: |
+          mkdir tmp_wiki
+          cd tmp_wiki
+          git init
+          git config user.name $USER_NAME
+          git config user.email $USER_EMAIL
+          git pull https://$USER_TOKEN@github.com/$OWNER/$REPOSITORY_NAME.wiki.git
```

Comment:
please make sure whether this one or the previous indentation is correct 🤔  as I can see this has 1-space and the below of this workflow, run steps has 2-spaces indentation